lecture2gether-vue/src/plugins/socket.io.ts

```diff
@@ -67,6 +72,11 @@ export const connect = (store: Store<any>) => {
         store.commit('setUrl', state.videoUrl)
     });
 
+    socket.on(receivedEvents.roomUserCountUpdated, (state: RoomUserCountEvent) => {
+        console.log("got event!")
+        store.commit('setUserCount', state.users);
+    });
+
```

Comment:
Please don't call the argument state since it is easy to confuse with the RoomState which also gets synchronized.
The log statement is also unnecessary.

```suggestion
    socket.on(receivedEvents.roomUserCountUpdated, (event: RoomUserCountEvent) => {
        store.commit('setUserCount', event.users);
    });

```